GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesList.cs

```diff
@@ -414,9 +414,16 @@ private static void RepositoryContextAction(ToolStripItem menuItem, Action<Repos
 
         private static string ShortenText(string text, Font font, float maxWidth)
         {
+            var ellipsis = '…';
+            var width = TextRenderer.MeasureText(text, font).Width;
+            if (width < maxWidth)
```

Comment:
This should be `<=`, shouldn't it?

Reply:
I think `<` is fine. Possibly even `< maxWidth - somePadding`.